### DISCUSSION TITLE
can't find "kube_hpa_status_condition" metrics

### DISCUSSION THREAD

**Discussion by namening** [prometheus/prometheus#9565] [Q&A]
![image](https://user-images.githubusercontent.com/41912290/138425561-bcfaceb1-8d8c-4de1-8ba4-28f663767d40.png)
hello,everyone
Why can't I find metrics about kubernetes hpa?
kube-state-metrics verison is 1.8.0
kurbenetes version is 1.15
prometheus version is 2.30

**Comment by namening** [comment]
who konw？

**Reply by argadepp** [reply]
facing similar issue, do we have a resolution for the same?

**Comment by argadepp** [comment]
Can you please answer ?

**Reply by argadepp** [reply]
facing similar issue, do we have a resolution for the same?
